src/librustc_data_structures/bitvec.rs

```diff
@@ -17,14 +17,15 @@ pub struct BitVector {
 }
 
 impl BitVector {
+    /// Create a new bitvector of at least num_bits length.
     pub fn new(num_bits: usize) -> BitVector {
         let num_words = u64s(num_bits);
         BitVector { data: vec![0; num_words] }
     }
 
     pub fn contains(&self, bit: usize) -> bool {
         let (word, mask) = word_mask(bit);
-        (self.data[word] & mask) != 0
+        (self.data.get(word).cloned().unwrap_or(0) & mask) != 0
```

Comment:
> It seems to me that asking for an index that doesn't exist should panic and that this might mask bugs in other code. 

It had such a behaviour for sizes which are not `0 mod 64` up to next `0 mod 64` bit already. This makes the behaviour consistent for all sizes.